tests/server/routes/auth.test.js

```diff
@@ -16,3 +16,13 @@ test('post /api/v1/auth/register', () => {
       expect(res.statusCode).toBe(201)
     })
 })
+
+test('post /api/v1/auth/logn', () => {
+  return request(server)
+    .post('/api/v1/auth/login')
+    .send({username: 'notfoo'})
+    .set('Accept', 'application/json')
+    .then(res => {
+      expect(res.statusCode).toBe(500)
```

Comment:
A `500` status code is a server-side error. I don't think this is what we should be expecting.